### ISSUE TITLE
100% cpu usage

### ISSUE THREAD

**Issue by colmaengus** [Haivision/srt#893]
I'm seeing a problem where CPU usage goes up to 100% for sender srt-live-transmit as soon as the sender and receiver connect.
I'm using docker to create two containers, one for sender, one for receiver (see below for details)
I'm using latest from master as of 30-Sept-2019.

**Dockerfile**

```
FROM ubuntu:18.04

ARG DEBIAN_FRONTEND=noninteractive

RUN apt-get update \
    && apt-get upgrade --no-install-recommends -y software-properties-common tclsh pkg-config cmake libssl-dev build-essential \
    && rm -rf /var/lib/apt/lists/*
```

**docker-compose.yaml**
```
version: '2'
services:
  sender:
    image: srt-dev
    working_dir: /usr/src/srt
    volumes:
      - ../srt:/usr/src/srt
    command: bash -c "tail -f /dev/null"
  receiver:
    image: srt-dev
    working_dir: /usr/src/srt
    volumes:
      - ../srt:/usr/src/srt
    command: bash -c "tail -f /dev/null"
```
After building srt-dev and starting the stack I shell into each container and run the following
**receiver**
`./srt-live-transmit srt://:8000 udp://host.docker.internal:9002`

**sender**
`./srt-live-transmit udp://localhost:9000 srt://receiver:8000`

As soon as they connect the sender container cpu goes up to 100%.
The full setup includes sending a udp stream from ffmpeg to the sender and connecting ffplay to the receiver but they don't have an impact on the high CPU.

**Comment by ethouris** [comment]
What's the characteristics of the stream? How reproducible is this? We'll try to repro this in our lab, as long as the data you left here suffice.

**Comment by colmaengus** [comment]
The stream I'm using is sintel-1024-surround.mp4. The command I'm using
`ffmpeg -re -i sintel-1024-surround.mp4 -c copy -f mpegts udp://localhost:9000?pkt_size=1316`
Adding ffmpeg to the container requires a small addition.
```
RUN add-apt-repository ppa:jonathonf/ffmpeg-4 \
    && apt-get update \
    && apt-get install --no-install-recommends -y ffmpeg \
    && rm -rf /var/lib/apt/lists/*
```
However the high cpu usage occurs as soon as the srt handshake is complete even without any input stream.
You should be able to reproduce what I'm seeing using the Dockerfile, docker-compose.yaml and commands above.

**Comment by gmadeo** [comment]
I am experiencing this with the 1.4 release. I see jitter is much improved, but CPU usage is much too high at 100%. 

I see this on Ubuntu 14, Ubuntu 18 and Debian 10.

Stream is nothing special, 8Mbps TS. Simply doing UDP > SRT and SRT > UDP.

**Comment by maxlovic** [comment]
Should be fixed by PR #919. Please check the latest master and reopen if the issue is still in force.
